app/pages/lab-pages-editor/helpers/convertKeyToIndex.js

```diff
@@ -0,0 +1,14 @@
+/*
+Transforms "T1234" (string) to 1234 (number).
+Returns 0 by default.
+ */
+/* eslint-disable radix */
+
+import { TASK_KEY_PREFIX, STEP_KEY_PREFIX } from './constants.js';
+
+export default function convertKeyToIndex(taskKeyOrStepKey = '') {
+  // regular expression looks like /^(?:T|P)(\d+)$/ - only the '\d+' is captured.
+  const re = RegExp(`^(?:${TASK_KEY_PREFIX}|${STEP_KEY_PREFIX})(\\d+)$`);
```

Comment:
This seems complicated, and will break on existing transcription workflows, where the keys begin with `S`. Why not slice the first character from each string instead?

Reply:
Yeah, this is over-engineered, and actually breaks when the index is more than 1 digit long. (e.g. P1234 = 4) Let me simplify this.